sdk/core/Azure.Core.Experimental/src/Variant/Variant.cs

```diff
@@ -951,6 +951,20 @@ public Variant(ArraySegment<char> segment)
         public static explicit operator decimal?(in Variant value) => value.As<decimal?>();
         #endregion
 
+        #region String
+        /// <summary>
+        /// TBD.
```

Comment:
docs?

Reply:
All of these cast operators currently say TBD - I'll definitely do a pass over them before long, but would like to do it as a holistic step rather than a one-off.